code/model/Feedback.php

```diff
@@ -1,21 +1,29 @@
 <?php
 class Feedback extends DataObject
 {
-	static $db = array(
+	/**
+	 * @var array
+	 */
+	public static $db = array(
 		'Rating' => 'Boolean',
 		'Comment' => 'Text',
-		'URL' => 'Text'
+		'URL' => 'Text',
+		'IP' => 'Text'
 	);
 
-	static $has_one = array(
+	/**
+	 * @var array
+	 */
+	public static $has_one = array(
 		'Page' => 'SiteTree'
 	);
 
-
-	static $summary_fields = array(
+	/**
+	 * @var array
+	 */
+	public static $summary_fields = array(
 		'Rating',
 		'Comment',
 		'URL'
 	);
```

Comment:
Oh, sure. I don't want to worry about the CMS side of things right now though. This round of work is just about the front-end and integration.